src/dynsight/_internal/analysis/rdf.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from ovito.io import import_file
```

Comment:
So, you are importing ovito here, and if you are not installing ovito with the test in the github actions, it will not be able to import this and fail.